### PR TITLE
fix backward compat support for tss lib v3

### PR DESCRIPTION
**Pull request by himanshuchawla009**
- Make tss lib types input types to be compatible with both v3 and v3 tss-dkls and tss frost libs

### CODE REVIEW

File: src/mpcCoreKit.ts [hunk 1]
```diff
@@ -43,12 +43,14 @@ import {
   InitParams,
   JWTLoginParams,
   MPCKeyDetails,
+  NewTSSLibType,
```
Comment (ieow) [inline src/mpcCoreKit.ts:46]: should we rename to v4TSSLibType to be more precise
as there will be always have new TssLibType

File: src/mpcCoreKit.ts [hunk 2]
```diff
@@ -312,20 +314,6 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
       await this.featureRequest();
       TorusUtils.setSessionTime(this.options.sessionTime);
     }
-
-    // if not redirect flow or session rehydration, ask for factor key to login
-
-    // dont wait for wasm to be loaded, we can reload it during signing if not loaded
```
Comment (ieow) [inline src/mpcCoreKit.ts:318]: why do we skip async pre-load?
this can reduce the time needed for the first signing

Comment (himanshuchawla009) [reply src/mpcCoreKit.ts:318]: matthias said, its just 30ms for loading wasm lib so i thought its fine to remove it for the sake of code clarity

File: src/mpcCoreKit.ts (hunk 1 above)
Comment (himanshuchawla009) [reply src/mpcCoreKit.ts:46]: alright can do this

Comment (himanshuchawla009) [reply src/mpcCoreKit.ts]: done

**Review by ieow — APPROVED** [review]
lgtm

**Pull request merged**